### PR TITLE
Introduce path directive

### PR DESCRIPTION
**Pull request by keiko713**
### Summary

With this PR, we introduce a new directive called PathDirective. Please check out README update for how to use it, but it's really similar to existing `@!model` directive.

Closes https://github.com/netlify/bitballoon/issues/10401

### Test plan

* [x] tests added or updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,6 +108,29 @@ class Accounts::OwnershipsController < ActionController::Base
 end
 ```
 
+### Standalone Path ###
+
+When one controller action method supports multiple paths, the additional path can be documented by using `@!path` directive.
+
+```ruby
+  # @!path [GET] /extra/accounts/ownerships/{id}
```
Comment (kschlarman) [inline README.md:118]: Can both of these paths contain the method description?

Comment (keiko713) [reply README.md:118]: I'm gonna poke around more to see if it's possible 👍

Comment (keiko713) [reply README.md:118]: just tested and confirmed that it works. so adding it as an example!

**Review by kschlarman — APPROVED** [review]
🌻

**Pull request merged**